app/js/deleteCategory.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * deletes category from database
+ * @param {number} id the category id
```

Comment:
Could this be reworded to be clearer?

Reply:
We have changed to just say category id - is that better?